service/matching/matching_engine.go

```diff
@@ -1915,6 +1918,7 @@ func (e *matchingEngineImpl) updatePhysicalTaskQueueGauge(pm *physicalTaskQueueM
 		metrics.NamespaceTag(pmImpl.ns.Name().String()),
 		metrics.TaskTypeTag(physicalTaskQueueParameters.taskType.String()),
 		metrics.PartitionTypeTag(physicalTaskQueueParameters.partitionType.String()),
+		metrics.VersionedTag(versioned),
```

Comment:
Is versioned coming from user provided value? Would metrics cardinality be a concern?

Reply:
no this can have only 3 values: "unversioned", "buildID", "versionSet"